protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -344,6 +344,14 @@ service TradingDataService {
     option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {tags: "Markets"};
   }
 
+  // List successor markets
+  //
+  // Given a market ID, return the full lineage of markets since inception, or all successor markets since and including
+  // the given market ID.
```

Comment:
Can we say something about the order of the markets in the returned array so users know what to expect? Something (assuming it is this way around) like. "The returned markets will be ordered such that market in position `i` will be the parent of market `i+1`"?